Directory.Build.props

```diff
@@ -443,7 +443,6 @@
   <!-- Test properties for full build. -->
   <PropertyGroup Condition="'$(BuildAllProjects)' == 'true'">
     <GenerateFullCoverageReport Condition="'$(GenerateFullCoverageReport)' == ''">true</GenerateFullCoverageReport>
-    <Performance Condition="'$(Performance)' == ''">false</Performance>
```

Comment:
Is this because -buildtests will now build all test projects?

Reply:
What are you referring to exactly? There is no performance switch anymore as we now have a dedicated target for it, if you mean that.

Reply:
I see. Makes sense.